### PR TITLE
checking grammar on operations and ops accessing cluster page, updati…

### PR DESCRIPTION
**Pull request by msdolbey**
…ng terminology on same pages, checked the commands

## Jira Ticket
N/A - release docs testing

## Description of changes being made
updating docs with some grammar fixes and positioning, links, checked the code snippets

## Checklist
- [ ] Change all affected versions, if applicable (e.g. 1.13, 2.0, 2.1).
- [ ] Test all commands and procedures, if applicable.
- [ ] Create your PR against `staging`, not `master`. 
- [ ] Provide an estimated date for deploying the doc change. Note: Improvements or fixes can be merged ASAP. 
- [ ] Add [redirects](https://github.com/mesosphere/dcos-docs-site/wiki/Redirects) if you are moving a page.
- [ ] Add release date to Release Notes page in the following format: <Package> was released on <Day>, <Month> <Year> ie `Mesosphere® DC/OS™ 2.1.0 was released on 9, June 2020`

See the [contribution guidelines](https://github.com/mesosphere/dcos-docs-site/wiki/Contributing) for more information.

Actually done here for the konvoy repo for future us here: 
https://github.com/mesosphere/konvoy/pull/1776

### CODE REVIEW

File: pages/ksphere/konvoy/1.4/operations/index.md [hunk 1]
```diff
@@ -7,7 +7,7 @@ excerpt: Manage your Konvoy cluster and deployed applications using platform ser
 enterprise: false
 ---
 
-After you deploy a Konvoy cluster and the platform services you want to use, you are ready to beginning managing cluster operations and its application workloads to optimize your organization's productivity.
+After you deploy a Konvoy cluster and the platform services you want to use, you are ready to begin managing cluster operations and its application workloads to optimize your organization's productivity.
```
Comment (russurquhart1) [inline pages/ksphere/konvoy/1.4/operations/index.md:10]:
```suggestion
After you deploy a Konvoy cluster and the platform services you want to use, you can begin managing cluster operations and its application workloads to optimize your organization's productivity.
```

File: pages/ksphere/konvoy/1.5/operations/accessing-the-cluster/index.md [hunk 2]
```diff
@@ -34,27 +36,31 @@ konvoy get ops-portal
...
 This UI is hosted on a web application within the cluster, which runs on the cluster's ingress load balancer.
 
+To access the UI:
+
 1. Enter that URL in your browser.
 2. Click the "Launch Console" button
 3. Enter your username and password as noted above, and click "Login".
 
-You will then see Konvoy's operations portal, which offers an overview of cluster status, and shortcuts to several dashboards to addon services such as Grafana.
+You will then see Konvoy's operations portal, which offers an overview of cluster status, and shortcuts to several dashboards to addon services, such as Grafana.
```
Comment (russurquhart1) [inline pages/ksphere/konvoy/1.5/operations/accessing-the-cluster/index.md:45]:
```suggestion
You will the Konvoy's operations portal. The portal offers an overview of cluster status, and shortcuts to several dashboards to addon services, such as Grafana.
```

File: pages/ksphere/konvoy/1.5/operations/accessing-the-cluster/index.md [hunk 3]
```diff
@@ -34,27 +36,31 @@ konvoy get ops-portal
...
 
-<p class="message--important"><strong>IMPORTANT: </strong>When using an automated provisioner, you should not delete the load balancer created by the cloud provider for the traefik service. If the load balancer is deleted, you can follow the steps below to regain access to the ops-portal.</p>
+<p class="message--important"><strong>IMPORTANT: </strong>When using an automated provisioner, you should not delete the load balancer created by the cloud provider for the traefik service. If the load balancer is deleted, you can follow the steps below to regain access to the ops-portal. In order to complete the below instructions, you need to update your kubectl credentials with the `admin.conf` file created for your Konvoy cluster by running the `./konvoy apply kubeconfig` command. More on that <a href="#configure-kubectl-for-cluster-administrators">here.</a></p>
```
Comment (russurquhart1) [inline pages/ksphere/konvoy/1.5/operations/accessing-the-cluster/index.md:47]:
```suggestion
<p class="message--important"><strong>IMPORTANT: </strong>When using an automated provisioner, do not  delete the load balancer created by the cloud provider for the traefik service. If the load balancer is deleted, follow the steps below to regain access to the ops-portal. In order to complete these instructions, you must update your kubectl credentials with the <code>admin.conf</code> file created for your Konvoy cluster by running the <code>./konvoy apply kubeconfig</code> command. More on that <a href="#configure-kubectl-for-cluster-administrators">here.</a></p>
```

File: pages/ksphere/konvoy/1.5/operations/accessing-the-cluster/index.md [hunk 4]
```diff
@@ -71,12 +77,12 @@ For example, you can use `kubectl` to:
...
 - View logs and status messages
 
-For a complete list of `kubectl` operations, see [Overview of kubectl](https://kubernetes.io/docs/reference/kubectl/overview/).
+For a complete list of `kubectl` operations, see [overview of kubectl][kubectl_overview].
 
 ### Install kubectl
 
-The specific steps required in install kubectl depend on your operating system platform.
-For platform-specific instructions to help you install kubectl, see the appropriate Kubernetes [installation and setup information][0] for the platform you use.
+The specific steps required in installing kubectl depend on your operating system platform.
```
Comment (russurquhart1) [inline pages/ksphere/konvoy/1.5/operations/accessing-the-cluster/index.md:84]:
```suggestion
The steps required to install kubectl depends on your operating system platform.
```

File: pages/ksphere/konvoy/1.5/operations/index.md [hunk 5]
```diff
@@ -7,7 +7,7 @@ excerpt: Manage your Konvoy cluster and deployed applications using platform ser
 enterprise: false
 ---
 
-After you deploy a Konvoy cluster and the platform services you want to use, you are ready to beginning managing cluster operations and its application workloads to optimize your organization's productivity.
+After you deploy a Konvoy cluster and the platform services you want to use, you are ready to begin managing cluster operations and its application workloads to optimize your organization's productivity.
```
Comment (russurquhart1) [inline pages/ksphere/konvoy/1.5/operations/index.md:10]:
```suggestion
After you deploy a Konvoy cluster and the platform services you want to use, you can begin managing cluster operations and its application workloads to optimize your organization's productivity.
```

File: pages/ksphere/konvoy/1.5/operations/accessing-the-cluster/index.md (hunk 2 above)
Comment (msdolbey) [reply pages/ksphere/konvoy/1.5/operations/accessing-the-cluster/index.md:45]: I think it's more clear the other way for this one

File: pages/ksphere/konvoy/1.5/operations/accessing-the-cluster/index.md [hunk 6]
```diff
@@ -71,12 +77,12 @@ For example, you can use `kubectl` to:
...
 - View logs and status messages
 
-For a complete list of `kubectl` operations, see [Overview of kubectl](https://kubernetes.io/docs/reference/kubectl/overview/).
+For a complete list of `kubectl` operations, see [overview of kubectl][kubectl_overview].
 
 ### Install kubectl
 
-The specific steps required in install kubectl depend on your operating system platform.
-For platform-specific instructions to help you install kubectl, see the appropriate Kubernetes [installation and setup information][0] for the platform you use.
+The steps required to install kubectl depends on your operating system platform.
```
Comment (msdolbey) [inline pages/ksphere/konvoy/1.5/operations/accessing-the-cluster/index.md:84]:
```suggestion
The steps required to install kubectl depend on your operating system platform.
```

**Comment by samvantran** [comment]
Can we make these changes in Konvoy repo instead? That is the process for Konvoy docs

**Comment by msdolbey** [comment]
pushed a pr to konvoy too.

**Pull request merged**